config/fcos/v1_6_exp/validate.go

```diff
@@ -54,7 +54,7 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 
 func isMountpointForbidden(path string) bool {
 	// Check if the path is a restricted directory
-	restrictedDirs := []string{"/boot", "/sys", "/usr", "/var", "/", "/sysroot/mnt"}
+	restrictedDirs := []string{"/boot", "/sys", "/usr", "/var", "/", "/sysroot/mnt", "/opt"}
```

Comment:
This commit looks like it should be apart of the original.

Reply:
And wdyt about making these alphabetical?

Reply:
ill keep that in mind the next time i use this approach!